.kivrc

```diff
@@ -51,5 +51,8 @@
 # User
 #ADDITIONAL_USER=${USER}
 
+# Path to custom USERDATA
+#USERDATA=
```

Comment:
Let's rename to `CUSTOM_USERDATA`.  See comment below.

Reply:
Ok 👍